Sources/Styleguide/DesignSystems/Layout.swift

```diff
@@ -1,30 +1,40 @@
 import Css
 import Prelude
 
+public enum Dimension: String {
+  case x
+  case y
+}
+
 extension Class {
-  public static let layout = (
-    fit: CssSelector.class("fit"),
-    inline: CssSelector.class("inline"),
-    block: CssSelector.class("inline"),
-    inlineBlock: CssSelector.class("inline"),
-    overflowHidden: CssSelector.class("overflow-hidden"),
-    overflowScroll: CssSelector.class("overflow-scroll"),
-    overflowAuto: CssSelector.class("overflow-auto"),
-    clearFix: CssSelector.class("clear-fix"),
-    left: CssSelector.class("left"),
-    right: CssSelector.class("right")
-  )
+  public enum layout {
```

Comment:
I just converted this back to an `enum` namespace from a tuple. for a moment there I was like the tuple style, but it can be a lil annoying and you can't document each field.